spec/std/isa/csr/Ssaia/sieh.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2025 Kallal Mukherjee
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: sieh
+long_name: Supervisor Interrupt Enable High
+address: 0x114
+priv_mode: S
+length: 32
+base: 32
+writable: true
+description: |
+  Upper 32 bits of sie.
+definedBy: Ssaia
+fields: {}
```

Comment:
The bits in `sieh` are aliases of the high-order bits of `sie`, but we don't seem to have `sie` defined yet. It's coming in PR #958. You might consider rebasing on that PR's branch, so you can add the aliases now.